src/App.vue

```diff
@@ -216,28 +199,10 @@ export default {
   methods: {
     ...mapActions(['setKeycloakRoles', 'setAuthRoles', 'setBusinessEmail', 'setBusinessPhone',
       'setBusinessPhoneExtension', 'setCurrentDate', 'setEntityName', 'setEntityType', 'setEntityStatus',
-      'setEntityBusinessNo', 'setEntityIncNo', 'setLastPreLoadFilingDate', 'setEntityFoundingDate', 'setLastAgmDate',
+      'setEntityBusinessNo', 'setEntityIncNo', 'setLastPreLoadFilingDate', 'setEntityFoundingDate',
       'setNextARDate', 'setTasks', 'setFilings', 'setRegisteredAddress', 'setRecordsAddress', 'setDirectors',
       'setLastAnnualReportDate', 'setConfigObject', 'setNameRequest']),
 
-    /** Stores today's date and sets timeout to keep it updated. */
```

Comment:
This didn't work if the browser tab (or computer) went to sleep, and I didn't like the idea of the date just changing in the background while the user may be doing who-knows-what.

So I now set the date each time the data is fetched (ie, dashboard reload). This will keep the date constant until the dashboard is reloaded.

I think this either approach is ideal, but I prefer the consistency of the second approach.